### PR TITLE
Updated Step 3 in README.rst

### LINKED ISSUE #251: Developer installation instructions should describe -e option
The installation instructions for developers say to do `pip install .` or `pip install -e .` without explaining the difference between the two. I think the instructions should either remove the first option or make it clear what each option does.

https://github.com/fury-gl/fury/blob/master/docs/source/installation.rst

### PR DESCRIPTION
**Pull request by tushar5526**
Fixes #251

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -60,11 +60,13 @@ Development
 
     pip install -r requirements/default.txt
 
-**Step 3.** Install fury via::
+**Step 3.** Install fury 
+
+As `Local Project Install <https://pip.pypa.io/en/stable/reference/pip_install/#id44>`__ using::
```
Comment (skoudoro) [inline README.rst:65]: Can you use lowercase?  `local project install`

File: README.rst [hunk 2]
```diff
@@ -60,11 +60,13 @@ Development
...
 
     pip install .
 
-or::
+Or as `“Editable” Installs <https://pip.pypa.io/en/stable/reference/pip_install/#id44>`__ using::
```
Comment (skoudoro) [inline README.rst:69]: same as above (lowercase)

File: README.rst (hunk 1 above)
Comment (tushar5526) [reply README.rst:65]: sure thing !

File: README.rst [hunk 3]
```diff
@@ -60,11 +60,13 @@ Development
 
     pip install -r requirements/default.txt
 
-**Step 3.** Install fury via::
+**Step 3.** Install fury 
+
+As `local project install <https://pip.pypa.io/en/stable/reference/pip_install/#id44>`__ using::
```
Comment (dmreagan) [inline README.rst:65]: As a local project installation

File: README.rst [hunk 4]
```diff
@@ -60,11 +60,13 @@ Development
...
 
     pip install .
 
-or::
+Or as `"editable" installs <https://pip.pypa.io/en/stable/reference/pip_install/#id44>`__ using::
```
Comment (dmreagan) [inline README.rst:69]: Or as an "editable" installation

Also say something like "If you are developing FURY, you most likely want this option."

Comment (tushar5526) [reply README.rst]: Did that, please take a look !

**Comment by skoudoro** [comment]
Thank you @tushar5526, merging

**Pull request merged**